packages/target-nodejs-sdk/test/proxy.spec.js

```diff
@@ -0,0 +1,51 @@
+const fetch = require("node-fetch");
```

Comment:
```js
const fetch = require("node-fetch");
```
I don't think this is necessary, is it?  Jest is configured to already have fetch.  I don't think you need the dependency in package.json either.  Try it at let me know.